packages/x-charts/src/context/ZAxisContextProvider.tsx

```diff
@@ -98,6 +98,15 @@ ZAxisContextProvider.propTypes = {
   zAxis: PropTypes.arrayOf(
     PropTypes.shape({
       colorMap: PropTypes.oneOfType([
+        PropTypes.shape({
```

Comment:
@alexfauquette this order changed in a lot of files and I don't know why 😓

Reply:
Weird. That's more likely an error in the TS to proptypes script that do not fix the order of elements. Nothing to worry too much